scripts/test-local.sh

```diff
@@ -7,12 +7,20 @@ if [ -z "$CORS_ORIGIN" ]; then export CORS_ORIGIN="http://foo,http://bar"; fi;
 
 set -u
 
-glob=$*
-if [ -z "$glob" ]; then
-  glob="test/local test/remote"
+GLOB=$*
+if [ -z "$GLOB" ]; then
+  GLOB="test/local test/remote"
+fi
+
+DEFAULT_ARGS="-R dot --recursive"
+
+# When running under Windows Subsystem for Linux,
+# some tests take unusually long to complete.
+if uname -a | grep --quiet 'Microsoft'; then
+  DEFAULT_ARGS="$DEFAULT_ARGS --timeout 5000"
```

Comment:
This is a little extra helper for me when running the tests; it's in a separate commit for easy removal if anyone feels like it's not appropriate here.

Reply:
Lol.

No, it's good. If it's useful for you, I'm sure it will be useful for other Windows contributors too!